webui.py

```diff
@@ -385,17 +386,20 @@ def __init__(self, handler, env):
         self.usercookie = None
         self.failed = None # error message if initialization already produced a failure
 
+        self.package_bucket = None
+        if self.config.database_aws_access_key_id and self.config.database_aws_secret_access_key:
+            self.s3conn = boto.s3.connect_to_region(
+                "us-west-2",
+                aws_access_key_id=self.config.database_aws_access_key_id,
+                aws_secret_access_key=self.config.database_aws_secret_access_key,
+            )
 
-        self.s3conn = boto.s3.connect_to_region(
-            "us-west-2",
-            aws_access_key_id=self.config.database_aws_access_key_id,
-            aws_secret_access_key=self.config.database_aws_secret_access_key,
-        )
-
-        self.package_bucket = self.s3conn.get_bucket(
-            self.config.database_files_bucket,
-            validate=False,
-        )
+            self.package_bucket = self.s3conn.get_bucket(
+                self.config.database_files_bucket,
+                validate=False,
+            )
+        else:
+            print("AWS database not fully configured. Getting the package files will be unavailable.", file=sys.stderr)
```

Comment:
Note that this is mostly indenting, so viewing the diff with [`?w=1` ](https://github.com/pypa/pypi-legacy/pull/536/files?w=1) might make things easier to review.